docs/xcm/integration/zombienet-testing.md

```diff
@@ -0,0 +1,107 @@
+
+The following instructions will explain how to setup & configure local _Shibuya - Shiden test network_.
+
+### Basic Setup Instructions
+
+For users who already know what they are doing, please check [this](https://github.com/AstarNetwork/Astar/tree/master/third-party/zombienet) folder in `Astar` repository for _ready-to-use_ `zombienet` configurations.
+
+1. For the sake of simplicity, prepare a folder called `zombienet` into which **ALL** binaries and config files will be placed.
+
+2. Download `zombienet` binary appropriate for your operating system (or install it in any way you prefer).
+
+3. Download `polkadot` binary and `astar-collator` binary. They can be found as part of release in official [polkadot](https://github.com/paritytech/polkadot/releases) and [Astar](https://github.com/AstarNetwork/Astar/releases) repositories.
+
+4. Use the configuration file `multi_parachains.toml` which can be found [here](https://github.com/AstarNetwork/Astar/tree/master/third-party/zombienet). Make sure to check the file to get a sense of which parameters are being used.
+
+5. Start the network. This is a command example: `./zombienet-linux -p native spawn multi_parachains.toml`. However, you can start the network in your preferred way (e.g. using `podman` or `kubernetes`).
+
+6. Lots of useful information will be printed, like commands used to generate chain specifications and commands used to start the network (useful for extracting RPC ports). At the end, a table with direct links to node's `polkadot-js page` will be created. Use this to interact with chain from your browser.
+
+7. After a minute or two, block production on both `Shibuya` and `Shiden` should start. This will usually happen after **relay-chain** reaches block 11, which will trigger a new session. HRMP channels will automatically be configured between the parachains (check the configuration file).
+
+8. The test network is running and users can interract with relay chain and both parachains. It is possible to deploy EVM and WASM smart contracts, send XCM instructions and everything else possible on live chains. In addition, users have direct access to `Alice` account, which has `sudo` privileges.
+
+> We will provide automated way of performing these setup actions in the future.
+
+### Basic Cross-Chain Assets Setup Instructions
+
+After completing the previous steps, a local test network with two parachains is running.
+The following steps will explain how to setup basic cross-chain payable assets for both parachains.
+
+The aim is to configure a cross-chain **SDN** asset on `Shibuya` and cross-chain **SBY** asset on `Shiden`. Both assets will be configured as _payable_, meaning they can be used to pay for XCM execution on remote chain.
+
+For all steps, encoded call data will be provided to simplify the process for the user. **ALL** calls should be executed as `Alice`.
```

Comment:
I think, a couple of words on how to actually use that encoded data would be good.

Reply:
I added a short sentence now - this is intended for devs however.
It is expected that they know about this or are able to investigate on their own. It's nothing specific to _our_ tools.